cmd/controller/main.go

```diff
@@ -128,7 +129,8 @@ func main() {
 	agonesInformerFactory := externalversions.NewSharedInformerFactory(agonesClient, 30*time.Second)
 	kubeInformationFactory := informers.NewSharedInformerFactory(kubeClient, 30*time.Second)
 
-	c := gameservers.NewController(wh, health, minPort, maxPort, sidecarImage, alwaysPullSidecar, kubeClient, kubeInformationFactory, extClient, agonesClient, agonesInformerFactory)
+	gsController := gameservers.NewController(wh, health, minPort, maxPort, sidecarImage, alwaysPullSidecar, kubeClient, kubeInformationFactory, extClient, agonesClient, agonesInformerFactory)
+	gssController := gameserversets.NewController(wh, kubeClient, extClient, agonesClient, agonesInformerFactory)
```

Comment:
@markmandel The one letter difference between variable names is easy to miss. How about `gs` and `gsSet` or something that is easier to parse quickly?

Reply:
I like gsSet better. I'm actually going to change that across the whole codebase. I've had issues writing this mixing up `gs` and `gss` and it took me ages to fix the bugs!